cmd/main.go

```diff
@@ -166,18 +166,38 @@ func run(ctx context.Context, logger *logrus.Logger) error {
 		return err
 	}
 
+	if cfg.ReportFormat == report.NoneFormat {
+		return nil
+	}
+
+	includePayloads := cfg.IncludePayloads
+	if cfg.ReportFormat == report.HtmlFormat || cfg.ReportFormat == report.PdfFormat {
+		askForPayloads := true
+		if isIncludePayloadsFlagUsed || cfg.NoEmailReport || cfg.Email != "" {
```

Comment:
This part: `cfg.NoEmailReport || cfg.Email != ""`

1. If `NoEmailReport` is set, it means that we should `Save report locally` only and should not send a report to the e-mail
2. If `Email` is set (not empty), it means that we should (?) send a report to the e-mail

E.g the logical expression becomes something like this, which is quite strange (same):
`if not send report to the e-mail || if send report to the e-mail (e-mail not empty)`

Is it true, or did I miss something?

Reply:
Both the `cfg.NoEmailReport == true` and `cfg.Email != ""` mean that the user has explicitly chosen the GoTestWAF behavior. In these cases, the user doesn't want to be asked about sending report, and we assume that the user doesn't want use interactive mode. Therefore, if the user explicitly specified what to do with the payloads in the report or explicitly indicated whether he wants to send the report by email, then we use either his explicit choice (true or false) for the cfg.IncludePayloads option, or leave the default value. The default value cfg.IncludePayloads is false, and asking the user to change this choice in interactive mode is an additional way to make sure that the user doesn't really want to include the payloads into the report.

Reply:
Fixed - https://github.com/wallarm/gotestwaf/pull/175/commits/8e0bbf0ba4cae589f4426dba59b7386165b6255a